librec/src/test/java/librec/data/DenseMatrixTest.java

```diff
@@ -0,0 +1,302 @@
+package librec.data;
+
+import librec.data.DenseMatrix;
+import librec.data.DenseVector;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.lang.reflect.Array;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class DenseMatrixTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  /* testedClasses: DenseMatrix */
+
+  /*
+   * Test generated by Diffblue Deeptest.
```

Comment:
Did we decide to eliminate the line coverage comments from tests we plan to submit?

Reply:
If so, we should update the diffblue.yml in the cx repo.